cirq/google/engine/engine.py

```diff
@@ -360,6 +362,65 @@ def run_batch(
                                         description=job_description,
                                         labels=job_labels)
 
+    def run_calibration(
+            self,
+            layers: List['cirq.google.CalibrationLayer'],
+            program_id: Optional[str] = None,
+            job_id: Optional[str] = None,
+            processor_ids: Sequence[str] = (),
+            gate_set: Optional[sgs.SerializableGateSet] = None,
+            program_description: Optional[str] = None,
+            program_labels: Optional[Dict[str, str]] = None,
+            job_description: Optional[str] = None,
+            job_labels: Optional[Dict[str, str]] = None,
+    ) -> engine_job.EngineJob:
+        """Runs the supplied Circuits via Quantum Engine.Creates
+
+        This will combine each Circuit provided in `programs` into
```

Comment:
This description appears to be obsolete - there are no 'programs' or 'params_list'.

Reply:
Fixed.